src/SFA.DAS.ApplyService.Web.UnitTests/Services/RefreshTrusteeServiceTests.cs

```diff
@@ -0,0 +1,300 @@
+            {
+                CharityNumber = charityNumber,
+                Trustees = listOfTrustees,
+                Status = "registered"
+            };
+
+            _qnaApiClient.Setup(x => x.GetQuestionTag(_applicationId, RoatpWorkflowQuestionTags.UKPRN)).ReturnsAsync(ukprn.ToString());
+            _qnaApiClient.Setup(x => x.GetQuestionTag(_applicationId, RoatpWorkflowQuestionTags.UKRLPVerificationCharityRegNumber)).ReturnsAsync(charityNumber);
+
+            _organisationApiClient.Setup(x => x.UpdateTrustees(ukprn.ToString(), listOfTrustees, It.IsAny<Guid>())).ReturnsAsync(true);
+
+            var currentStatuses = new List<RoatpApplicationStatus>();
+            currentStatuses.Add(new RoatpApplicationStatus { ApplicationId = _applicationId, Status = ApplicationStatus.InProgress });
+            _applicationApiClient.Setup(x => x.GetExistingApplicationStatus(ukprn.ToString())).ReturnsAsync(currentStatuses);
+            _outerApiClient.Setup(x => x.GetCharityDetails(charityNumberValue)).ReturnsAsync(charity).Verifiable();
+
+            var updateResponse = new SetPageAnswersResponse { ValidationPassed = true };
+            _qnaApiClient.Setup(x => x.UpdatePageAnswers(_applicationId, RoatpWorkflowSequenceIds.YourOrganisation, RoatpWorkflowSectionIds.YourOrganisation.WhosInControl, It.IsAny<string>(), It.IsAny<List<Answer>>())).ReturnsAsync(updateResponse);
+
+            var result = await _service.RefreshTrustees(_applicationId, _userId);
+
+            var redirectResult = result as RefreshTrusteesResult;
+            redirectResult.CharityNumber.Should().Be(charityNumber);
+            redirectResult.CharityDetailsNotFound.Should().BeFalse();
+
+            _organisationApiClient.Verify(x => x.UpdateTrustees(ukprn.ToString(), listOfTrustees, It.IsAny<Guid>()), Times.Once);
+            _qnaApiClient.Verify(x => x.UpdatePageAnswers(_applicationId, RoatpWorkflowSequenceIds.YourOrganisation, RoatpWorkflowSectionIds.YourOrganisation.WhosInControl, RoatpWorkflowPageIds.WhosInControl.CharityCommissionTrustees, It.IsAny<List<Answer>>()), Times.Once);
+            _qnaApiClient.Verify(x => x.ResetPageAnswersBySequenceAndSectionNumber(_applicationId, RoatpWorkflowSequenceIds.YourOrganisation, RoatpWorkflowSectionIds.YourOrganisation.WhosInControl, RoatpWorkflowPageIds.WhosInControl.CharityCommissionTrustees), Times.Once);
+            _qnaApiClient.Verify(x => x.ResetPageAnswersBySequenceAndSectionNumber(_applicationId, RoatpWorkflowSequenceIds.YourOrganisation, RoatpWorkflowSectionIds.YourOrganisation.WhosInControl, RoatpWorkflowPageIds.WhosInControl.CharityCommissionTrusteesDob), Times.Once);
+            _qnaApiClient.Verify(x => x.ResetPageAnswersBySection(_applicationId, RoatpWorkflowSequenceIds.CriminalComplianceChecks, RoatpWorkflowSectionIds.CriminalComplianceChecks.CheckOnWhosInControl), Times.Once);
+        }
+
+        [Test]
+        public void refresh_trustees_and_update_trustees_fails_exception_thrown()
```

Comment:
Missing test for when you are raising error on update of organisation.

Reply:
The update trustee call updates the organisation.